packages/ui-common/src/components/RoomPlayer/PlayerMenu.tsx

```diff
@@ -154,6 +160,13 @@ function PlayerMenu({
           </Tooltip>
         </Stack>
       </Paper>
+      {finished != null && roomStartContext != null && finished && (
```

Comment:
!==? why did the linter not catch this?

Reply:
`!= null` is acceptable because it's checking whether or not the value is defined (e.g. it is short hand for `val !== null && val !== undefined`). Doing this is a better alternative than doing `if (val)` because val may be `false` or a falsy value